subprojects/plugins/src/integTest/groovy/org/gradle/api/tasks/bundling/JarIntegrationTest.groovy

```diff
@@ -26,6 +27,7 @@ import java.util.jar.Manifest
 
 class JarIntegrationTest extends AbstractIntegrationSpec {
 
+    @TestReproducibleArchives
```

Comment:
We could also annotate the individual test methods. Which tests shouldn't have `@TestReproducibleArchives`?